### PR TITLE
MAT-7415 remove Saved Codes header from Saved Codes tab

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-7415](https://jira.cms.gov/browse/MAT-7415)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/CqlBuilderPanel/codesSection/codesSubSection/savedCodesSubSection/SavedCodesSubSection.tsx [hunk 1]
```diff
@@ -357,119 +357,112 @@ export default function SavedCodesSubSection({
 
   return (
     <div>
-      <ExpandingSection
```
Comment (adongare) [inline src/CqlBuilderPanel/codesSection/codesSubSection/savedCodesSubSection/SavedCodesSubSection.tsx:360]: this is deleted. should the import be deleted as well if not being used?

File: src/CqlBuilderPanel/codesSection/CodesSection.test.tsx [hunk 2]
```diff
@@ -112,7 +112,7 @@ describe("CodesSection", () => {
     const savedCodesSectionButton = await screen.findAllByRole("button");
     const savedCodesSectionHeading = within(
       savedCodesSectionButton[0]
-    ).getByText("Saved Codes");
-    expect(savedCodesSectionHeading).toBeInTheDocument();
+    ).queryByText("Saved Codes");
+    expect(savedCodesSectionHeading).not.toBeInTheDocument();
```
Comment (adongare) [inline src/CqlBuilderPanel/codesSection/CodesSection.test.tsx]: we probably do not need to check for the thing that is not part of the functionality anymore

Comment (sb-cecilialiu) [reply src/CqlBuilderPanel/codesSection/CodesSection.test.tsx]: @adongare Yeah sure will remove. Thanks!

File: src/CqlBuilderPanel/codesSection/codesSubSection/savedCodesSubSection/SavedCodesSubSection.tsx (hunk 1 above)
Comment (sb-cecilialiu) [reply src/CqlBuilderPanel/codesSection/codesSubSection/savedCodesSubSection/SavedCodesSubSection.tsx:360]: @adongare Will do, thanks!

File: src/CqlBuilderPanel/codesSection/CodesSection.test.tsx (hunk 2 above)
Comment (sb-cecilialiu) [reply src/CqlBuilderPanel/codesSection/CodesSection.test.tsx]: @adongare I have made those 2 changes, thanks!

**Review by adongare — APPROVED** [review]

**Pull request merged**